response_operations_ui/controllers/survey_controllers.py

```diff
@@ -30,3 +30,17 @@ def get_survey(short_name):
 
     logger.debug('Successfully retrieved survey', short_name=short_name)
     return response.json()
+
+
+def get_surveys_dict():
+    # TODO cache this dictionary in app
+    surveys_list = get_surveys_list()
+    return {survey['id']: survey for survey in surveys_list}
+
+
+def get_survey_short_name_by_id(survey_id):
+    try:
+        return get_surveys_dict()[survey_id]['shortName']
+    except KeyError:
+        logger.exception("failed to resolve survey short name", survey_id=survey_id)
+        return 'Unavailable'
```

Comment:
In the case of a KeyError would this result in the display of "Unavailable Team" in the UI. You might want to speak to UI about what to display in this instance.